src/yoroi-wallets/walletManager.ts

```diff
@@ -355,12 +355,15 @@ export class WalletManager {
     throw new Error('invalid wallet')
   }
 
-  closeWallet(): Promise<void> {
+  async closeWallet(): Promise<void> {
     if (!this._wallet) return Promise.resolve()
+    await this._wallet.clear()
```

Comment:
When a wallet is deleted or synced, it is closed.

Reply:
resync + remove must delete the /utxo storage

Reply:
`wallet.clear()` is deleting the utxo storage. And `walletManager.closeWallet` is used by `walletManager.removeWallet()` and `walletManager.resyncWallet()` methods.

Should I use `wallet.close()` directly on `walletManager.resyncWallet()` and `walletManager.removeWallet()`?

Reply:
closing the wallet must not clear the utxo cache, the only actions that must do it are: resync wallet and remove wallet
the UtxoService, will resume from where it was left when opening the wallet again, so instead of sync from the scratch, it will sync from the last block it's processed.

Reply:
why does closing a wallet delete storage?